reset_security_server.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Script should be run as sudo
+# Usage reset_security_server.sh -n=server(s) -h=path/to/hosts.txt -a=path/to/ansible
+# Usage example: reset_security_server.sh -n=ss3,ss4 -h=../../X-Road/ansible/hosts/lxd_hosts.txt -a=../../X-Road/ansible
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+    exit 1
+fi
+
+for i in "$@"
+do
+case $i in
+    -n=*|--name=*)
+    NAME="${i#*=}"
+    shift
+    ;;
+    -h=*|--hosts=*)
+    HOSTS="${i#*=}"
+    shift
+    ;;
+    -a=*|--ansible=*)
+    ANSIBLE="${i#*=}"
+    shift
+    ;;
+    *)
+        echo "Usage: reset_security_server.sh -n=name(s) -h=hosts.txt -a=ansible_folder"
+        exit 1
+    ;;
+esac
+done
+
+names=$(echo "$NAME" | tr "," "\n")
+container_list=$(lxc list --format=json | jq -r '.[] | .name + ","')
+containers=$(echo "$container_list" | tr "," "\n")
+
+for name in $names
+do
+    for container in $containers
+    do
+        if [ $name = $container ]
+        then
+            printf "\n Deleting LXD container %s\n" "$name"
+            lxc delete "$name" -f
+        fi
+    done
+done
+
+cd $ANSIBLE
+printf "\nRunning Ansible script: %s/ansible-playbook -i %s xroad_init.yml\n" "$ANSIBLE" "$HOSTS"
+ansible-playbook -i "$HOSTS" xroad_init.yml
```

Comment:
These file names we depend on could also get their own constants, that way we can more easily change them (this file name is already used in one ``printf`` directly before)

Reply:
Probably also early existence check for ``xroad_init.yml`` could be in order, but that might be unnecessary, I have not had a chance yet to see how long or understandable of an error message is produced when it is not present where it is supposed to.

EDIT: ok, the error message on missing file seems clear enough as is, but the script goes on and acts as if something  was actually done, session run:

```
$  ./reset_security_server.sh -n=asd -h=asd -a=asd 
./reset_security_server.sh: line 49: cd: asd: No such file or directory

Running Ansible script: asd/ansible-playbook -i asd xroad_init.yml
ERROR! the playbook: xroad_init.yml could not be found

 Ansible script finished installing security servers 

Creating api-key for LXD container asd
Error: not found
Error: not found
```

After which I was wondering if it actually created ``asd`` container somehow, but no, ``lxc list`` shows that no new containers have been added.

Reply:
Added some checks, so the existence of the directory and Ansible script file is checked before running it